### PR TITLE
Write blog post about Honeybadger Incoming Error API.

### PR DESCRIPTION
**Pull request by anastasiosl**
https://thinkneardev.atlassian.net/browse/REX-3061

### CODE REVIEW

**Comment by anastasiosl** [comment]
@eduardog you might want to take a look at this, too.

File: source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+date: 2015-05-01 15:25:00 -0700
+author: Tassos Livogiannis
+comments: true
+is_newest: true
+categories:
+---
+
+Our systems at Thinknear run on a 24/7 basis and monitoring them for errors is essential to prevent production issues that can have a negative impact on our business.
+One of the tools we use for error reporting is [Honeybadger](https://www.honeybadger.io).
+When one of our systems sends an error (usually a Ruby or Java exception) it gets posted to Honeybadger website, engineers get an email notification and they can log into Honeybadger to see the details of the exception.
```
Comment (eduardog) [inline source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown]: "gets posted to _the_ Honeybadger website"?

File: source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+It is important for debugging to see all the relevant error information in Honeybadger, grouped in a useful way.
+
+Below is the summary portion of an error reported to Honeybadger.
+{% img center /images/honeybadger-post/error_summary.png 945 457 'image' 'images' %}
+
+Recently, we wanted to add information about the incoming request (both headers and body) and we had to look deeper into Honeybadger's [incoming error API](https://www.honeybadger.io/pages/collector) to identify where things show up in a Honeybadger error page.
+The [payload tester](https://app.honeybadger.io/notice/test) tool provided is great to test sample JSON requests and see how the error will be reported on the page.
+However, the sample page it shows does not contain sections Context and App Environment.
```
Comment (eduardog) [inline source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown]: "does not contain _the_ ~~sections~~ Context and App Environment _sections_"

File: source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+Furthermore, the documentation does not provide a mapping of JSON keys and values to elements on the resulting error page so we had to reverse engineer the format of the page, based on test requests.
+
+In this blog post our findings are shared and we hope they are useful to other engineers who are using [Honeybadger](https://www.honeybadger.io) for error reporting.
```
Comment (eduardog) [inline source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown]: Is this a reasonable thing to be saying here? I don't actually know

**Comment by eduardog** [comment]
done with a pass.  Looks good to me? I'm not sure I understand how these are supposed to look.

File: source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown (hunk 3 above)
Comment (anastasiosl) [inline source/_posts/2015-04-30-reverse-engineering-honeybadger-incoming-error-request.markdown]: I could remove it. The resulting text (omitting that line) would still make sense.

**Comment by anastasiosl** [comment]
Updated.

**Pull request merged**